app/models/base/Model.ts

```diff
@@ -55,7 +55,7 @@ export default abstract class Model {
       const store = this.store.rootStore.getStoreForModelName(
         properties.relationClassResolver().modelName
       );
-      if ("fetch" in store) {
+      if ("fetch" in store && this[properties.idKey]) {
```

Comment:
This prevents an existing issue where `*.info` calls were being rejected as `400` because the `id` was being passed as `undefined | null`.

Reply:
Looks like this has recently been fixed already.